src/tests/routes/index.spec.js

```diff
@@ -1,6 +1,6 @@
 import chai from 'chai';
 import chaiHttp from 'chai-http';
-import app from '..';
+import app from '../..';
```

Comment:
why did you move the test file insde the tests / routes folder, do you intend for us to write tests separately for our controllers and models?

Reply:
That was how Anna specified it when set up the test, I was going to run it by you before I made the changes. I'm guess we'll be testing models and controllers, maybe